### ISSUE TITLE
carousel: ensure accessibility [31 July]

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#1738]
Related:

- Reassignment to our team: https://pluralsight.slack.com/archives/C70HPSJPP/p1620146354081200
- Best descriptions of issues we have 23-26: https://docs.google.com/spreadsheets/d/1ht9c5utZQ1yvp7qX0eK2Pm2DenRyJADXyUXjSHgwkRM/edit#gid=891834841
- Possibly more description: https://docs.google.com/document/d/1QHYr63CDN9n4EGDvN6SCoiKfNVpFNivg/edit#heading=h.gjdgxs

Verify that:

- [ ] Press next/previous, the page is announced in screenreader
- [ ] Can read the card titles in active page with screenreader
- [ ] Arrow keys work for paging and are announced the same as next/previous presses

**Comment by jaketrent** [comment]
Committed to fixing or verifying by 31 July.

**Comment by jaketrent** [comment]
Fixed by #1769.

Verification results were different because the implementation of Carousel changed so much.

- [x] Now the carousel reads like a normal list. There are no pages. You tab between each.
- [x] All card titles are readable, as they would be in an `ul > li`.
- [x] There is now not paging per se. Tabbing for keyboard users or next element on screenreaders will advance to the next item. Arrow keys are supported, but they are only for visual scrubbing. Focus, and thus screenreaders/keyboard users, are not affected.

**Comment by jaketrent** [comment]
Reported done to Amy as of today.
